src/runtime_src/core/tools/common/XBUtilities.cpp

```diff
@@ -241,6 +241,24 @@ str2index(const std::string& str, bool _inUserDomain)
   return bdf2index(str, _inUserDomain);
 }
 
+void
+XBUtilities::xrt_xbutil_version_cmp()
+{
+  boost::property_tree::ptree pt_xrt;
+  xrt_core::get_xrt_info(pt_xrt);
+  boost::property_tree::ptree empty_ptree;
+
+  std::string xrt_version = pt_xrt.get<std::string>("version", "N/A");
+  const boost::property_tree::ptree& available_drivers = pt_xrt.get_child("drivers", empty_ptree);
+  for(auto& drv : available_drivers) {
+    const boost::property_tree::ptree& driver = drv.second;
+    std::string drv_name = driver.get<std::string>("name", "N/A");
```

Comment:
Hmm...this causes the error message to be rather crypted by stating the version is "N/A".  Consider something a little more useful.  Possible "unknown".
```suggestion
    std::string drv_name = driver.get<std::string>("name", "<unknown>");
```